ci/tests.py

```diff
@@ -82,6 +94,7 @@ def test_generate_report(self):
             'http://hub/registrations/',
             json={
                 'count': 1,
+                'next': None,
```

Comment:
This won't actually test that we're getting the next page successfully. Maybe we should have 2 results split over 2 pages, and ensure that we get both results in the export.